src/Mimi.jl

```diff
@@ -721,6 +722,38 @@ end
 
 #End of graph section
 
-include("references.jl")
+# Begin plotting section
+
+"""
+Three defaults: single line plot, multiple line plots with legend, bar graph
+when indexing over regions, etc.
+"""
+function auto_plot(m::Model, component::Symbol, parameter::Symbol ; index::Symbol = :time, legend::Symbol = nothing, xlabel = string(index), ylabel = string(parameter), dest = nothing)
```

Comment:
Can we add a method to the ``plot`` function from the ``Plots`` package here? Just define this as ``Plots.plot``.